src/mono/mono/mini/aot-compiler.c

```diff
@@ -5265,6 +5274,20 @@ MONO_RESTORE_WARNING
 		}
 	}
 
+	if (acfg->aot_opts.export_symbols_outfile && export_symbols->len) {
+		char *export_symbols_out = g_string_free (export_symbols, FALSE);
+		acfg->export_symbols_outfile = fopen (acfg->aot_opts.export_symbols_outfile, "w");
+		g_free (acfg->aot_opts.export_symbols_outfile);
```

Comment:
other aot_opts in acfg holds there value through the lifetime of the AOT compiler execution, I think we should do the same with export_symbols_outfile.

Reply:
Thanks!